timemachine/md/minimizer.py

```diff
@@ -145,12 +147,18 @@ def minimize_host_4d(mols, host_system, host_coords, ff, box, mol_coords=None) -
     x0 = combined_coords
     v0 = np.zeros_like(x0)
 
-    x0 = fire_minimize(x0, u_impls, box, np.ones(50))
-    # context components: positions, velocities, box, integrator, energy fxns
-    ctxt = custom_ops.Context(x0, v0, box, intg, u_impls)
-    _, xs, _ = ctxt.multiple_steps(np.linspace(1.0, 0, 1000))
+    u_impls = bind_potentials(parameterize_system(hgt, ff, 1.0))
+    x = fire_minimize(x0, u_impls, box, np.ones(50))
+
+    for lamb in np.linspace(1.0, 0, 50):
+        u_impls = bind_potentials(parameterize_system(hgt, ff, lamb))
+        # context components: positions, velocities, box, integrator, energy fxns
+        ctxt = custom_ops.Context(x, v0, box, intg, u_impls)
+        _, xs, _ = ctxt.multiple_steps(lamb * np.ones(50))
+        x = xs[-1]
```

Comment:
did we want to carry over the velocities?  (Currently it resets them to zero each time)

Reply:
In offline discussion, decided to leave as-is; preserving velocities between lambda windows seems to decrease efficiency, requiring more windows to pass the check. Commented in e154a22).